docs/sources/writing-guide/reuse-directories/index.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Reuse directories of content with Hugo mounts
+keywords:
+  - content reuse
+  - shared content
+  - Hugo
+menuTitle: Reuse directories of content
+title: Reuse directories of content with Hugo mounts
+---
+
+# Reuse directories of content with Hugo mounts
+
+> **Note:** The following task can only be performed by a Grafana Labs employee.
+
+Hugo mounts mount a source directory at a destination directory before the Hugo build.
```

Comment:
```suggestion
Before Hugo performs a build, _Hugo mounts_ mount a source directory at a destination directory.
```
Italicizing this because it is otherwise odd to read.